
string agv_name

# Product type
string movable_tray_name

string movable_tray_type

# Target pose
geometry_msgs/Pose pose